pest/src/parser_state.rs

```diff
@@ -1098,7 +1098,7 @@ impl<'i, R: RuleType> ParserState<'i, R> {
     }
 }
 
-fn constrain_idxs(start: i32, end: Option<i32>, len: usize) -> Option<std::ops::Range<usize>> {
+fn constrain_idxs(start: i32, end: Option<i32>, len: usize) -> Option<::std::ops::Range<usize>> {
```

Comment:
Thanks for the PR! :) How about importing the range up top?

Reply:
I just realized that `Range` was already imported at the top of the file. Whoops! :)